docs/pages/components/tile.md

```diff
@@ -10,186 +10,911 @@ summary:
+        <div class="fd-tile__background-img" style="background-image: url('https://placeimg.com/200/200/nature')"></div>
+        <button aria-label="toggle play button" class="fd-tile__toggle"></button>
+        <div class="fd-tile__container">
+            <div class="fd-tile__header">
+                <div class="fd-tile__title">Wind Map: Monitoring Real-Time</div>
+                <div class="fd-tile__subtitle">SAP News</div>
+            </div>
+            <div class="fd-tile__footer">
+                <span class="fd-tile__footer-text">May 14, 2020</span>
+            </div>
+            <div class="fd-tile__page-indicator">
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot fd-tile__dot--active"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="docs-section-container">
+    <div tabindex="0" class="fd-tile fd-tile--double fd-tile--slide">
+        <div class="fd-tile__background-img" style="background-image: url('https://placeimg.com/400/200/nature')"></div>
+        <<button aria-label="toggle pause button" class="fd-tile__toggle fd-tile__toggle--pause"></button>
+        <div class="fd-tile__container">
+            <div class="fd-tile__header">
+                <div class="fd-tile__title">Wind Map: Monitoring Real-Time and Forecasted Wind Conditions across the Globe</div>
+                <div class="fd-tile__subtitle">SAP News</div>
+            </div>
+            <div class="fd-tile__footer">
+                <span class="fd-tile__footer-text">May 14, 2020</span>
+            </div>
+            <div class="fd-tile__page-indicator">
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot fd-tile__dot--active"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+                <span class="fd-tile__dot"></span>
+            </div>
+        </div>
+    </div>
+    <div tabindex="0" class="fd-tile fd-tile--double fd-tile--s fd-tile--slide">
+        <div class="fd-tile__background-img" style="background-image: url('https://placeimg.com/400/200/nature')"></div>
+        <<button aria-label="toggle pause button" class="fd-tile__toggle fd-tile__toggle--pause"></button>
+        <div class="fd-tile__container">
+            <div class="fd-tile__header">
+                <div class="fd-tile__title">Wind Map: Monitoring Real-Time and Forecasted Wind Conditions across the Globe</div>
+                <div class="fd-tile__subtitle">SAP News</div>
+            </div>
+            <div class="fd-tile__footer">
+                <span class="fd-tile__footer-text">May 14, 2020</span>
+            </div>
```

Comment:
All this text should have `text-shadow` for accessibility (color contrast) reasons. This is also according to spec:
![image](https://user-images.githubusercontent.com/43764832/82719275-ed93cf00-9c5d-11ea-9988-173d790af0e4.png)

Reply:
Hi @prsdthkr the font color and the text-shadow use calculations which are not themable. We've contacted the team working on the sap-theming and we also logged an issue https://github.com/SAP/fundamental-styles/issues/1048